frontend/src/pages/Agreement.tsx

```diff
@@ -116,13 +127,49 @@ const Agreement: FC = () => {
                 </Stack>
               </AccordionSummary>
               <AccordionDetails sx={{ backgroundColor: '#333842' }}>
-                <Stack spacing={2}>
+                <Stack spacing={0}>
                   {chapter?.sections?.map((section) => (
-                    <SectionCard
-                      key={section.id}
-                      section={section}
-                      onClick={() => navigate(`section/${section.id}`)}
-                    />
+                    <div key={section.id}>
+                      <SectionCard
+                        section={section}
+                        onClick={() => navigate(`section/${section.id}`)}
+                      />
+                      <Divider id="devider" textAlign="center" variant="fullWidth">
```

Comment:
- [ ] Use class instead of id (since there are many dividers)
- [ ] typo: devider -> divider

Reply:
fixed